www-client/qutebrowser/qutebrowser-9999.ebuild

```diff
@@ -30,10 +31,14 @@ RDEPEND="${COMMON_DEPEND}
 	>=dev-python/jinja-2.8[${PYTHON_USEDEP}]
 	>=dev-python/pygments-2.1.3[${PYTHON_USEDEP}]
 	>=dev-python/pypeg2-2.15.2[${PYTHON_USEDEP}]
-	dev-python/PyQt5[${PYTHON_USEDEP},declarative,gui,network,printsupport,webkit,widgets]
+	dev-python/PyQt5[${PYTHON_USEDEP},declarative,gui,network,printsupport,webengine?,webkit?,widgets]
 	dev-python/sip[${PYTHON_USEDEP}]
 	>=dev-python/pyyaml-3.12[${PYTHON_USEDEP}]
 	gstreamer? ( dev-qt/qtwebkit:5[gstreamer] )
+	webengine? (
```

Comment:
Could you elaborate on this? qutebrowser explicitly requires these versions to use webengine.